### PR TITLE
Update Feature Toggles Documentation

### PR DESCRIPTION
**Pull request by pvaliani**
Resolves NEWSWORLDSERVICE-1488

**Overall change:**
- Updates Toggles documentation
- Updates Simorgh Runbook with Toggles Config API info

**Code changes:**

N/A 

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project
- [ ] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/lib/config/toggles/README.md [hunk 1]
```diff
@@ -1,3 +1,27 @@
...
+   const cachedResponse = cache && cache.get(url);
+  if (cachedResponse) {
+    return {
+      ...defaultLocalToggles,
+      ...cachedResponse,
+    };
+  }
+```
+
+If a toggle is not configured in iSite or in the local toggle configs coupled to Simorgh, then the toggle value will default to false.
```
Comment (andrewscfc) [inline src/app/lib/config/toggles/README.md]:
```suggestion
If a toggle is not configured in iSite or in the local toggle configs deployed with Simorgh, then the toggle value will default to false.
```
Slight wording preference, what do you think?

File: src/app/lib/config/toggles/README.md [hunk 2]
```diff
@@ -1,3 +1,27 @@
...
+
+```
+   const cachedResponse = cache && cache.get(url);
+  if (cachedResponse) {
+    return {
+      ...defaultLocalToggles,
+      ...cachedResponse,
+    };
+  }
+```
```
Comment (andrewscfc) [inline src/app/lib/config/toggles/README.md]: can you add a link to the code around here to help maintainers in the future?

File: src/app/lib/config/toggles/README.md (hunk 1 above)
Comment (pvaliani) [reply src/app/lib/config/toggles/README.md]: Yeah agree, all good!

File: src/app/lib/config/toggles/README.md (hunk 2 above)
Comment (pvaliani) [reply src/app/lib/config/toggles/README.md]: Yep was going to do that just forgot to grab the permalink. Done!

File: src/app/hooks/useToggle/README.md [hunk 3]
```diff
@@ -1,9 +1,8 @@
 # useToggle Hook
 
-## Feature Toggles
+### Feature Toggles
 
-Feature toggles are designed to be global (i.e not service specific) and are dependant on the application environment.
-Feature toggles can be found in `src/app/lib/config/toggles`
+A description of how feature toggles work can be found in `/src/app/lib/config/toggles/README.md`
```
Comment (eagerterrier) [inline src/app/hooks/useToggle/README.md]: can we make that a link to https://github.com/bbc/simorgh/blob/latest/src/app/lib/README.md

**Review by eagerterrier — CHANGES_REQUESTED** [review]
nice. Thanks. Can we just add a link for convenience?

File: src/app/hooks/useToggle/README.md (hunk 3 above)
Comment (pvaliani) [reply src/app/hooks/useToggle/README.md]: Yes, absolutely. Done!

**Review by eagerterrier — APPROVED** [review]
lgtm-ship-it

**Review by nicola-d-bbc — APPROVED** [review]

**Pull request merged**